dawn/examples/python/global_index_stencil.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# ===-----------------------------------------------------------------------------*- Python -*-===##
+#                          _
+#                         | |
+#                       __| | __ ___      ___ ___
+#                      / _` |/ _` \ \ /\ / / '_  |
+#                     | (_| | (_| |\ V  V /| | | |
+#                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+#
+#
+#  This file is distributed under the MIT License (MIT).
+#  See LICENSE.txt for details.
+#
+# ===------------------------------------------------------------------------------------------===##
+
+"""Copy stencil HIR generator
```

Comment:
I think this is a hold-over from copying the file. We should update this to reflect the global index example.

Reply:
done